### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by abdoElHodaky** [adonisjs/core#2679] [Ideas]
Is there future releases support graph databases and its similarities? 
May add theses future to next release
Thanks for your interest

**Comment by avidcoder123** [comment]
What is a graph db

**Reply by abdoElHodaky** [reply]
Graph-based database like Cassandra or neo4j 
Thanks for your interest

**Comment by RomainLanz** [comment]
Hey @abdoElHodaky! 👋 

What are you calling "support graph databases"?
In the end, your AdonisJS application is a Node.js application. So feel free to install whatever modules you need to access whatever database you need.

It is not because the core team has not created a module for a specific thing that you cannot do that thing.
